src/RA_Dlg_Achievement.cpp

```diff
@@ -515,11 +515,13 @@ INT_PTR Dlg_Achievements::AchievementsProc(HWND hDlg, UINT nMsg, WPARAM wParam,
                             }
                         }
                     }
-                    else if (MessageBox(hDlg,
-                        TEXT("Are you sure that you want to download fresh achievements from ") RA_HOST_URL TEXT("?\n")
-                        TEXT("This will overwrite any changes that you have made with fresh achievements from the server.\n"),
-                        TEXT("Refresh from Server"),
-                        MB_YESNO | MB_ICONWARNING) == IDYES)
+                    else if (auto msg = []() noexcept {
+                        tstring tstr{ TEXT("Are you sure that you want to download fresh achievements from ") };
+                        tstr.append(RA_HOST_URL);
+                        tstr.append(TEXT("?\n"));
+                        tstr.append(TEXT("This will overwrite any changes that you have made with fresh achievements from the server.\n"));
+                        return tstr;
+                    }; MessageBox(hDlg, msg().c_str(), TEXT("Refresh from Server"), MB_YESNO | MB_ICONWARNING) == IDYES)
                     {
```

Comment:
Some of you guys might not like this, but this was the most optimal way I could thing of.